courses/free-courses-ja.md

```diff
@@ -11,4 +11,6 @@
 
 ### Scratch
 
+* [How to make Mochi Master Game in scratch](https://www.youtube.com/watch?v=1cCth98Y2Lo) - Coding With Chris
```

Comment:
This is a good tutorial but the content is in English.

Reply:
I'm wondering about video's category, is video is a course or some else category?